websites/R/Roblox Forum/presence.ts

```diff
@@ -0,0 +1,61 @@
+const presence = new Presence({
+	clientId: "993929577099448481",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "ig-mal",
+			smallImageKey: "small",
+			smallImageText: "Swooosh!",
+		},
+		date = Math.floor(Date.now() / 1000);
+
+	if (document.location.pathname === "/") {
+		presenceData.details = "Viewing the homepage";
+		presenceData.startTimestamp = date;
```

Comment:
`startTimestamp` only needs to be assigned once and can be done within the `presenceData` object

Reply:
Okay, I'll be doing this once I'm done eating